app/src/main/java/com/sunasterisk/weather/utils/Constant.kt

```diff
@@ -6,4 +6,7 @@ object Constant {
     const val BASE_URL = "https://api.darksky.net/forecast/"
     const val BASE_API_KEY = BuildConfig.WEATHER_API_KEY + "/"
     const val COMMA = ","
+    const val LATITUDE_DEFAULT = 42.3601
+    const val LONGITUDE_DEFAULT = -71.0589
```

Comment:
Cái này là toạ độ gì đây?
Chú thích vào, nếu mà chỉ để hiển thị tạm thời thì xoá đi